Stack/Opc.Ua.Core/Types/Utils/Utils.cs

```diff
@@ -1331,9 +1331,19 @@ public static string ToHexString(byte[] buffer)
 
             StringBuilder builder = new StringBuilder(buffer.Length * 2);
 
-            for (int ii = 0; ii < buffer.Length; ii++)
+            if (bigEndian)
```

Comment:
Big endian is depending on the "size" of the variable, just invert the array may not be enough